### PR TITLE
Update all SVGs & add remaining services

### PR DESCRIPTION
**Pull request by sareh**
Part of #635

**Overall change:** Adds SVGs for Arabic, Pashto and Urdu services. Updates SVGs for Igbo, News, Pidgin, Thai, Yoruba. 

**Code changes:**

- Adds SVGs for Arabic, Pashto and Urdu services (Ensured these were optimised using an online tool)
- Updates snapshot tests
- Updates SVGs for Igbo, News, Persian, Pidgin, Thai, Yoruba so that they match the new SVGs in size and formatting of the `BBC News |`. 
- Added SVGs for these other services: afaanoromoo, afrique, amharic, azeri, bengali, burmese, gahuza, gujarati, hausa, hindi, indonesia, korean, kyrgyz, marathi, nepali, russian, serbian, sinhala, somali, swahili, tamil, telegu, tigrinya, turkish, ukchina, ukranian, vietnamese, zhongwen 

Storybook stories: http://localhost:8180/?path=/story/utilities-svgs--arabic
Screen Shot Arabic SVG
http://localhost:8180/?path=/story/utilities-svgs--pashto
Screen Shot Pashto SVG
http://localhost:8180/?path=/story/utilities-svgs--urdu
Screen Shot Urdu SVG

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by greenc05** [comment]
You might need to look at the fills @sareh https://github.com/bbc/psammead/issues/699 Will the associated visually hidden text be added in another PR?

**Comment by sareh** [comment]
@greenc05 Thank you, Catharine! I'll make that change. Yes visually hidden text is added in the Brand component. This component purely has the SVGs - e.g. [the Igbo one here](https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-assets/src/svgs/igbo.jsx)

File: packages/utilities/psammead-assets/CHANGELOG.md
Comment (sareh) [inline packages/utilities/psammead-assets/CHANGELOG.md]:
```suggestion
| 1.2.0   | [PR#1523](https://github.com/bbc/psammead/pull/1523) Add SVGs for these services: afaanoromoo, afrique, amharic, arabic, azeri, bengali, burmese, gahuza, gujarati, hausa, hindi, igbo, indonesia, korean, kyrgyz, marathi, nepali, news, pashto, persian, pidgin, russian, serbian, sinhala, somali, swahili, tamil, telegu, thai, tigrinya, turkish, urdu, ukchina, ukranian, vietnamese, zhongwen. Update these existing SVGs: igbo, news, persian, pidgin, thai, yoruba.  |
```

**Comment by sareh** [comment]
**Accessibility testing these SVGs**

In Firefox, changing the colours in preferences:
Screen Shot Firefox colours BBC Arabic

On a Windows PC, with IE11, using high contrast mode:

Arabic logo (an example of a new logo) with High Contrast version 1
windows-ie11-highcontrast-arabic

Arabic Logo with High Contrast version 2
windows-ie11-highcontrastv2-arabic

Arabic Logo with High Contrast version 3
windows-ie11-highcontrastv3-arabic

Yoruba logo (an example of an updated logo)
windows-ie11-highcontrast-yoruba

For comparison's sake, here is the current Pidgin site logo in two different high contrast palettes:
current pidgin site
current pidgin site v2


Notes here for how these were carried out: https://bbc-news.github.io/accessibility-news-and-you/accessibility-news-and-testers#checklist---manual

**Review by jamesdonoh — APPROVED** [review]
I won't pretend I've scrutinised all the SVG code very closely :-) but all looks generally sensible - thanks!

**Review by dr3 — APPROVED** [review]
LGTM assuming a11y review before merge

**Review by 12 — APPROVED** [review]

**Comment by sareh** [comment]
@dr3 The UX and A11y reviews were carried out prior to code review for this ticket, since there was quite a bit of back and forth and updates to the SVG markup. Results of A11y review are in this comment: https://github.com/bbc/psammead/pull/1523#issuecomment-518188928

**Comment by paruchurisilpa** [comment]
LGTM..

**Pull request merged**
